src/MIDebugEngine/AD7.Impl/AD7BoundBreakpoint.cs

```diff
@@ -65,7 +74,7 @@ public int Delete()
         // Called by the debugger UI when the user is enabling or disabling a breakpoint.
         int IDebugBoundBreakpoint2.Enable(int fEnable)
         {
-            _enabled = fEnable == 0 ? false : true;
+            Enabled = fEnable == 0 ? false : true;
```

Comment:
Right, we need to disable the bound breakpiont with something like "disable 2.1", 
For this change I'm only renaming the member. I'll add a TODO here and log a bug. And fix this in a separate change

---
In reply to: [70909937](https://github.com/Microsoft/MIEngine/pull/383#discussion_r70909937) [](ancestors = 70909937)